lib/ember-cli/app.rb

```diff
@@ -339,7 +339,7 @@ def env_hash
 
     def exec(cmd, method: :system)
       Dir.chdir root do
-        Kernel.public_send(method, env_hash, cmd, err: :out)
+        Kernel.public_send(method, env_hash, cmd, err: :out) or fail(cmd)
```

Comment:
@geoffharcourt yup, ended up using that [here](b8b8963e3e754dc52b8c2992dbbff19d8cb498b4)